packages/bff/src/services/authorizationService.ts

```diff
@@ -239,6 +239,41 @@ export function authorizationServiceBuilder(
 
       const claims = buildSupportClaims(selfcareId, tenant);
 
+      const { serialized: sessionToken } =
+        await interopTokenGenerator.generateSessionToken(
+          claims,
+          config.supportLandingJwtDuration
+        );
+
+      return sessionToken;
+    },
+    generateJwtFromSaml: async (
+      correlationId: string,
+      samlResponse: string,
+      tenantId: string
+    ): Promise<string> => {
+      validateSamlResponse(samlResponse);
+
+      const { serialized } =
+        await interopTokenGenerator.generateInternalToken();
+
+      const headers = {
+        "X-Correlation-Id": correlationId,
+        Authorization: `Bearer ${serialized}`,
+      };
+
+      const tenant = await tenantProcessClient.tenant.getTenant({
+        params: { id: config.pagoPaTenantId },
```

Comment:
Scala here uses `sAMLTokenRequest.tenantId`, which differs from what's done in the `samlLoginCallback` case called by the authorization router. Could you double check?